kgforge/specializations/resources/datasets.py

```diff
@@ -92,7 +92,7 @@ def add_files(self, path: str, content_type: str = None) -> None:
         _set(self, "hasPart", distribution)
 
     @catch
-    def download(self, source: str, path: str, overwrite: bool = False) -> None:
+    def download(self, path: str, source: str = "distributions", overwrite: bool = False) -> None:
```

Comment:
Hi @MFSY ! Did you switch the order of the arguments `path` and `source` deliberately? If so, we just need to remember to adjust the notebooks as well.

Reply:
Hi @annakristinkaufmann,

> Did you switch the order of the arguments path and source deliberately?

Yes, so that I can set a default value for `source` and enable users to just call `dataset.download(path="./")` works. By default the user does not need to know about how actual files are encoded in the json structure.

>If so, we just need to remember to adjust the notebooks as well.

Yes. [Done](https://github.com/BlueBrain/nexus-forge/pull/85/commits/f8e5d8daf44a4d3eba55b5f04dfb584e67959b4f).